docs/compression_algorithms/CompressWeights.md

```diff
@@ -6,17 +6,44 @@
 
 The Weights Compression algorithm is aimed at compressing the weights of the models and can be used to optimize the model footprint and performance of large models where the size of weights is relatively larger than the size of activations, for example, Large Language Models (LLM). The algorithm compresses weights only for Linear and Embedding layers.
 
+##### INT8 and NF4 modes.
+
+By default, weights are compressed to 8-bit integer data type - "INT8" mode.
+OpenVINO backend has also an experimental support for "NF4" mode - compression to [nf4](https://arxiv.org/pdf/2305.14314v1.pdf) data type.
+It goes with a grouped quantization, when small group of weights (e.g. 128) in the channel dimension share quantization parameters (scale).
+First embedding and last linear layers are always compressed to 8-bit integer data type in the "NF4" mode.
+Percent of the rest (internal) layers compressed to NF4 can be configured by "ratio" parameter.
+E.g. ratio=0.9 means 90% of internal layers compressed to nf4 and the rest to 8-bit integer data type.
 #### User guide
 
-- Compress weights of linear layers and embeddings to int8
+- Compress weights to 8-bit integer data type.
 
 ```python
 from nncf import compress_weights
 compressed_model = compress_weights(model)
 ```
 
+- Compress internal weights to nf4 data type with group size = 128.
```

Comment:
Why do you use 'internal' here and below but not in int8 case? Please align it.

Reply:
I see, but I believe it's still confusing. Why do you want to explicitly use 'internal'? I think it's enough to explain that first and last layers are quantized to int8

Reply:
Removed mentioning of internal weight to reduce confusion, explicitly say about first and last layers